packages/components/src/Progress/Progress.scss

```diff
@@ -1,5 +1,6 @@
 $tc-progress-color: $brand-success !default;
-$gradient-part-width: 100% / 6;
+$solid-gradient-part-width: 22%;
```

Comment:
Where does these values come from ? I think you should write the calculation you did from readability / maintenance reasons  (`$gradient-part-width: 100% / 6;`). It is preprocessed by SASS, so it doesnt affect performances.